docroot/modules/humsci/hs_paragraph_types/modules/hs_row/modules/hs_row_display/hs_row_display.module

```diff
@@ -8,6 +8,20 @@
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
 
+/**
+ * Implements template_preprocess_paragraph().
+ */
+function hs_row_display_preprocess_paragraph__hs_row(&$variables) {
+  $paragraph = $variables['paragraph'];
+
+  if (isset($paragraph->field_paragraph_style)) {
```

Comment:
this is not the proper way to get field data from an entity. see https://www.metaltoad.com/blog/drupal-8-entity-api-cheat-sheet for some quick references

Reply:
Good callout and thanks for the link. We'll look into this first thing tomorrow. 👍